dsync/dsync.go

```diff
@@ -37,14 +37,14 @@ var log = golog.Logger("dsync")
 
 const (
 	// default to parallelism of 3. So far 4 was enough to blow up a std k8s pod running IPFS :(
-	defaultPushParallelism = 3
+	defaultPushParallelism = 1
 	// default to parallelism of 3
 	// TODO (b5): tune this figure
-	defaultPullParallelism = 3
+	defaultPullParallelism = 1
 	// total number of retries to attempt before send is considered faulty
 	// TODO (b5): this number should be retries *per object*, and a much lower
 	// number, like 5.
-	maxRetries = 25
+	maxRetries = 80
```

Comment:
Since this value is total retries instead of per-object, this means that a small send, which can't successfully send for some reason, will take a long time to fail. For example, if I send something with only 1 block, that single block will retry 80 times.

I realize it's a larger change to fix this problem, but could we get TODOs explaining where the code needs to be modified to handle this? Would it be changing `sender.start` to handle the retries per block instead? Do replies from the receiving side have details about what went wrong? I'm thinking if the receiver is out of disk space (for example) or suddenly has invalid permissions then there's no reason to retry, as opposed to something like a network error.